docs/docs/tutorials/orca-yarn-tutorial.md

```diff
@@ -0,0 +1,532 @@
+                                    transforms.Normalize((0.5,), (0.5,))])
+
+    get_remote_file_to_local(remote_path="hdfs://url:port/path/to/dataset", local_path="/tmp/dataset")
+
+    trainset = torchvision.datasets.FashionMNIST(root="/tmp/dataset", train=True, download=False, transform=transform)
+
+    trainloader = torch.utils.data.DataLoader(trainset, batch_size=batch_size, shuffle=True, num_workers=0)
+    return trainloader
+```
+
+# 2. Prepare Environment
+Before running the BigDL program on YARN, you need to setup the environment following the steps below:
+
+## 2.1 Setup JAVA & Hadoop Environment
+You need to download and install JDK in the environment, and properly set the environment variable `JAVA_HOME`, which is required by Spark. JDK8 is highly recommended.
+
+### 2.1.1 Setup JAVA Environment
+```bash
+# For Ubuntu
+sudo apt-get install openjdk-8-jre
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/
+
+# For CentOS
+su -c "yum install java-1.8.0-openjdk"
+export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.282.b08-1.el7_9.x86_64/jre
+
+export PATH=$PATH:$JAVA_HOME/bin
+java -version  # Verify the version of JDK.
+```
+
+### 2.1.2 Setup Hadoop Environment
+Check the Hadoop setup and configurations of our cluster. Make sure we correctly set the environment variable `HADOOP_CONF_DIR`, which is needed to initialize Spark on YARN:
+```bash
+export HADOOP_CONF_DIR=/path/to/hadoop/conf
+```
+
+## 2.2 Install Python Libraries
+### 2.2.1 Install Conda
+You need first to use conda to prepare the Python environment on the __Client Node__ (where you submit applications). You could download and install Conda following [Conda User Guide](https://docs.conda.io/projects/conda/en/latest/user-guide/install/linux.html) or executing the command as below.
+```bash
+# Download Anaconda installation script 
+wget -P /tmp https://repo.anaconda.com/archive/Anaconda3-2020.02-Linux-x86_64.sh
+
+# Execute the script to install conda
+bash /tmp/Anaconda3-2020.02-Linux-x86_64.sh
+
+# Please type this command in your terminal to activate Conda environment
+source ~/.bashrc
+``` 
+
+### 2.2.2 Use Conda to Install BigDL and Other Python Libraries
+Create a conda environment, install BigDL and all the needed Python libraries in the created conda environment:
+``` bash
+# "env" is conda environment name, you can use any name you like.
+# Please change Python version to 3.6 if you need a Python 3.6 environment.
+conda create -n env python=3.7 
+conda activate env
+```
+Alternatively, You can install the latest release version of BigDL (built on top of Spark 2.4.6 by default) as follows:
+```bash
+pip install bigdl
+```
+You can install the latest nightly build of BigDL as follows:
+```bash
+pip install --pre --upgrade bigdl
+```
+
+__Note:__ 
+* Using Conda to install BigDL will automatically install libraries including `conda-pack`, `torch`, `torchmetrics`, `torchvision`, `pandas`, `numpy`, `pyspark==2.4.6`, and etc.
```

Comment:
it is strange to users to see torch and pandas related packages here.
Just change to including `pyspark==2.4.6` and other related dependencies?